data_hub_api/docmaps/sql/docmaps_index.sql

```diff
@@ -21,24 +23,49 @@ t_distinct_hypothesis_uri_doi_version_with_elife_doi_version AS (
     *,
     ROW_NUMBER() OVER(
       PARTITION BY source_doi
-      ORDER BY source_version
+      ORDER BY source_doi_version
     ) AS elife_doi_version
   FROM t_distinct_hypothesis_uri_doi_version
 ),
 
+t_distinct_hypothesis_uri_id_and_timestamp AS(
+  SELECT 
+    DISTINCT
```

Comment:
I believe we shouldn't need DISTINCT here as we are including the `hypothesis_id` of which there should only be one coming out of the hypothesis view?